packages/@coorpacademy-progression-engine/src/test/compute-next-step.microlearning.js

```diff
@@ -58,3 +59,13 @@ test('should return the extraLife endpoint when progressions has no more lives &
     type: 'node'
   });
 });
+
+test('should return the the fail endpoint when progressions has at least one extra life and extraLife has already been refused', t => {
```

Comment:
should return the failure endpoint when progression has at least one remaining extra life and extraLife has already just been refused